### DISCUSSION TITLE
Move timezones with me

### DISCUSSION THREAD

**Discussion by hellified** [iSoron/uhabits#2004] [Feature Requests]
As I travel I want my, for instance, 10 am reminders to be 10 am local time, not my home time zone.

**Comment by herrbrixter** [comment]
Related question. Do the values adjust with timezones?

Ex: I toggled Yes on Apr 3, 11 PM at my current timezone. Apr 3 is checked.

I moved to another timezone which is +2 hours. Does that mean the checkmark will remain in Apr 3 or move to Apr 4 (11 PM becomes next day's 1 AM)?

I hope the values stay fixed. In my example above, I prefer Apr 3 to remain checked while Apr 4 is unchecked.
